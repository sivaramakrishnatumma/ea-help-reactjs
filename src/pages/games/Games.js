import React from "react";

const Games = () => {
  return (
    <div className="page-container">
      <h1 className="center">Games Page</h1>
    </div>
  );
};

export default Games;
