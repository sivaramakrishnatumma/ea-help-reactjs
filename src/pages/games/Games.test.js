import React from "react";
import { shallow } from "enzyme";
import Games from "./Games";

describe("Games page test case", () => {
  it("should test Games Page", () => {
    const wrapper = shallow(<Games />);
    expect(wrapper).toMatchSnapshot();
  });
});
