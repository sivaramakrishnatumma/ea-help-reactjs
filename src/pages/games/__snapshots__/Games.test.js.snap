// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Games page test case should test Games Page 1`] = `
<div
  className="page-container"
>
  <h1
    className="center"
  >
    Games Page
  </h1>
</div>
`;
